package.json

```diff
@@ -46,5 +46,8 @@
     "husky": "^8.0.3",
     "lint-staged": "^13.2.2",
     "prettier": "^2.8.8"
+  },
+  "dependencies": {
```

Comment:
I understand that we don't use the twitter api sdk? If not please omit this part same for package.json

Reply:
Omit it here then do npm install to sync the package-lock.json

Reply:
good catch, thank you